internal/store/builder.go

```diff
@@ -221,6 +223,10 @@ func (b *Builder) buildLimitRangeStore() *metricsstore.MetricsStore {
 	return b.buildStore(limitRangeMetricFamilies, &v1.LimitRange{}, createLimitRangeListWatch)
 }
 
+func (b *Builder) buildMWCStore() *metricsstore.MetricsStore {
+	return b.buildStore(namespaceMetricFamilies, &v1.Namespace{}, createNamespaceListWatch)
```

Comment:
I'd prefer clarity to brevity here. Let's name it something like `buildMutatingWebhookConfigurationStore` or `buildMutatingWebhookConfigStore`.

Also, you should be creating a ListWatch method specific to your resource. Here you have re-used the list watch method of namespace.